scripts/install

```diff
@@ -34,29 +34,25 @@ ynh_mysql_generate_db "$user" "$app"
 ynh_system_user_create "$user" "$local_path"
 
 ynh_setup_source "$local_path" "$user"
-    
-ynh_configure config.php "$local_path/application/config/config.php"
 
-# Fill LimeSurvey database
-sed "s/\`prefix_/\`$prefix/g" $local_path/installer/sql/create-mysql.sql > ./structure.sql
-mysql -u $db_user -p$db_pwd $db_user < ./structure.sql
-ynh_configure data.sql ./data.sql
-mysql -u $db_user -p$db_pwd $db_user < ./data.sql
+ynh_configure config.php "$local_path/application/config/config.php"
 
 # Randomize Password user
 ls_cli=$local_path/application/commands/console.php
 set +x
-ynh_exec_as "$user" php $ls_cli resetpassword "$admin" "$(ynh_string_random 24)"
+ynh_exec_as "$user" php $ls_cli install "ynh_admin" "$(ynh_string_random 24)" "Administrator" "$admin@$domain"
 set -x
-
+# Setup^extra DB part (@todo : replace by a plugin system)
+ynh_configure data.sql ./data.sql
+mysql -u $db_user -p$db_pwd $db_user < ./data.sql
 # Set permissions
 ynh_set_default_perm $local_path
 sudo chmod -R u+w $local_path/tmp
 sudo chmod -R u+w $local_path/upload
-sudo chmod -R u+w $local_path/application/config/
+#~ sudo chmod -R u+w $local_path/application/config/
 
 sudo yunohost app addaccess $app -u $admin
-ynh_sso_access "/index.php?r=admin,/index.php?r=plugins,/scripts"
```

Comment:
I don't remember why I have put a protection on /index.php?r=plugins,/scripts
May be it's historical (with the last version of the package)